Contributing.md

```diff
@@ -8,6 +8,27 @@ yarn
 npm -i
 ```
 
+## Before You Begin
+
+### Ensure Nebula Logger Is Installed
```

Comment:
@renatoliveira thanks for submitting this! I would suggest some tweaks to the terminology here. For instance, if I'm not testing out the Nebula adapter, I don't always install Nebula Logger while working on scratch orgs.  I'd hardly say that it's a requirement; it's only required if you want to use source tracking. Therefore, I'd break this down into two different sections:

1. Source Tracking Enabled Sandboxes & Scratch Orgs
  a. Install the version of Nebula Logger specified in Apex Rollup's `sfdx-project.json` file, e.g.:

```bash
sf package install -p <package version 04t> -w 30 -r
```
2. Non-Source Tracking Enabled Sandboxes & Scratch Orgs (Where you don't use the `Push Source To Default Org...` commands)
  a. Use the `Deploy Source To Org` or CLI equivalents to deploy the `./rollup` directory, e.g.:

```bash
sf project deploy start --source-dir ./rollup
```
  b. Use the `Deploy Source To Org` or CLI equivalents to deploy the `./extra-tests` directory, e.g.:

```bash
sf project deploy start --source-dir ./extra-tests
```

That also means you can remove the `Whether you are developing on a sandbox or a new scratch org, please be sure to also deploy the `extra-tests` directory. ` line from the "Ensure All Rollup Files Are Installed" section

Reply:
>I don't always install Nebula Logger while working on scratch orgs.

For me it seemed like a requirement. How do you deploy the project to a scratch org? I cloned the repo and created an org and could not just push the source into it. The deployment fails with a missing reference to the Logger class. I assumed that this was due to it missing NL.

As for developing on sandboxes... Is there a reason nowadays for someone to develop an app in a sandbox instead of a scratch org?

Reply:
@renatoliveira I understand that it may seem like a requirement to you, but this is a monorepo where not everything _needs_ to be deployed in order to get up and running. I deploy the project to a scratch org in exactly the way I've detailed above, by either:

- installing Nebula Logger
- deploying the `rollup` and `extra-tests` directories

My commentary has no bearing on somebody's preference for development environment. While I use scratch orgs, I know that not everybody does. This is a strictly unopinionated library in that sense.

Reply:
I've made some changes according to the discussion. Does this look good for you?